### ISSUE TITLE
null is not an object (evaluation '_data2.default.ddp.on')

### ISSUE THREAD

**Issue by hakosl** [inProgress-team/react-native-meteor#32]
when using @connectMeteor i get the error: `null is not an object (evaluation '_data2.default.ddp.on')`, the problem seems to be in Mixin.js:123, when calling Data.ddp.on, because in Data.js, ddp is set to null, i don't really know what is going on behind the scenes, but i would really like some help to get it working, I just tried out the example code.

**Comment by Mokto** [comment]
Hi, I just completely refractored Mixin.js friday, can you please update to 1.0.0-beta15 and tell me what line is wrong ?

Also, did you follow the [detailed installation instructions](https://github.com/inProgress-team/react-native-meteor/blob/master/docs/Install.md) to add decorators to babel ?

**Comment by Mokto** [comment]
Last thing, are you sure you called Meteor.connect at the top of your app ?

**Comment by vermiculite** [comment]
I had this issue with 1.0.0-beta12 but having just updated to 1.0.0-beta17 the issue is resolved.

**Comment by hakosl** [comment]
I'm still having pretty much the same problem, it's just in a different file now, getting `null us not an object (evaluating 'this.ddp.on')`. Now the error is in data.js:20 onChange, it seems that for some reason. Data.ddp is still null, I don't know what is going on behind the scenes, and yes i added the decorators to my .babelrc and installed them with npm. this is my index.js file: 
```
import React, { View, Text, Component } from 'react-native';
import Meteor, { connectMeteor } from 'react-native-meteor';

@connectMeteor
export default class App extends Component {
  componentWillMount() {
    const url = 'http://192.168.56.1:3000/websocket';
    Meteor.connect(url);
  }

  getMeteorData() {
    const posts = Meteor.collections('posts').findOne();
    console.log(posts);
  }

  startMeteorSubscriptions() {
    Meteor.subscribe('posts');
  }

  renderRow(todo) {
    return (
      <Text>{todo.title}</Text>
    );
  }

  render() {
    return (<View>
      <Text>K</Text>
    </View>);
  }
}

```

**Comment by vermiculite** [comment]
@hakosl I guess you did already, but I had to restart the packager. Took me a while to figure that out.

**Comment by ko0stik** [comment]
Same as @hakosl here, I tried different combinations and all ended up the same way. The only way I found to circumvent that was to remove `getMeteorData`. I tried to display `Meteor.user()` from `render` in a `setTimeOut` and it was correct so connection is definitely there.

I browsed source code but I'm not sure where it goes south. I put a screenshot below.

![img_2243](https://cloud.githubusercontent.com/assets/5154713/13904342/984ba3fa-ee9d-11e5-9dac-88e2dfd3abd5.jpg)
